src/views/index.html.ejs

```diff
@@ -4,22 +4,39 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1" />
-    <meta name="description" content="Biofactoid is an app that captures the content of biological papers." />
-    <meta name="keywords" content="mentena, app, bio, biology, paper, academic, academia, publish, peer, review, data" />
+    <meta name="keywords" content="biofactoid, app, bio, biology, paper, academic, academia, publish, peer, review, data, summary, digital, profile, connect, connected" />
+    <meta name="robots" content="all" />
 
     <% if(typeof document !== typeof undefined && typeof TWITTER_ACCOUNT_NAME !== typeof undefined && typeof BASE_URL !== typeof undefined){ %>
+      <title><%= document.citation.title %> : Biofactoid</title>
+      <meta name="description" content="<%= document.text.replace(/\n/g, ' ') %> : <%= document.citation.title %>" />
+      <meta name="author" content="<%- document.citation.authors.authorList.map(author => `${author.name}`).join(', ') %>">
+      <script type="application/ld+json">
+      {
+        "@context" : "http://schema.org",
+        "@type" : "Article",
+        "name" : "<%= document.citation.title %>",
```

Comment:
Google says use `headline` but maybe that's for news article https://developers.google.com/search/docs/data-types/article#technical-guidelines

Reply:
We have both now

Reply:
ok